java/src/jmri/jmrit/beantable/signalmast/AddSignalMastPanel.java

```diff
@@ -221,8 +208,6 @@ public AddSignalMastPanel(SignalMast mast) {
         apply.setVisible(true);
         create.setVisible(false);
         
-        this.mast = mast;
```

Comment:
Why is this line removed? I might be missing something, but to me, this seems to be a functional change.

Reply:
`SignalMast mast;` was removed ( around line 56 ) as was not being read from anywhere, just being set in this 1 place so felt safe to remove.

Reply:
Thanks. I missed that.